txp/Pos/Txp/Logic/Local.hs

```diff
@@ -54,7 +56,8 @@ txProcessTransaction
 txProcessTransaction itw@(txId, txAux) = do
     let UnsafeTx {..} = taTx txAux
     tipDB <- GS.getTip
-    bootEra <- gsIsBootstrapEra
+    epoch <- siEpoch <$> getCurrentSlotBlocking
```

Comment:
I used `getCurrentSlotBlocking` here, but I don't know whether it's okay to block here.